### PR TITLE
Update search page styles

### PR DESCRIPTION
**Pull request by domoscargin**
https://trello.com/c/lu1zDbsy/267-2-are-we-ready-to-migrate-buyer-frontend-to-govuk-frontend-v3

Simplifying styles on the search pages in preparation of migration to DS3. This was a slightly trickier one to extricate from the migration branch, and has impacted functional tests: https://github.com/alphagov/digitalmarketplace-functional-tests/pull/818

### CODE REVIEW

**Comment by lfdebrux** [comment]
### My review checklist

I've:

- [ ] looked at the code
- [ ] tested locally
- [ ] run functional tests locally

File: app/assets/scss/_search-page.scss [hunk 1]
```diff
@@ -1,94 +1,60 @@
...
-  $lot-filters__gutter: 24px;
+/**
+ * Lot/category filter list
+ */
+.app-lot-filter {
 
-  .lot-filters {
+  a {
+    text-decoration: none;
+  }
```
Comment (lfdebrux) [inline app/assets/scss/_search-page.scss:13]: Is there any reason to remove the link underlines? I'm not certain but I'd worry about falling foul of [WCAG 2.1 failure 73](https://www.w3.org/WAI/WCAG21/Techniques/failures/F73)

File: app/templates/search/_categories_wrapper.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <div id="js-dm-live-search-categories" class="js-dm-live-search-fade">
-  <div class="lot-filters">
+  <div class="app-lot-filter">
```
Comment (lfdebrux) [inline app/templates/search/_categories_wrapper.html:2]: Thank you for adding `app-` to all these classes, makes my life easier when counting outstanding work 🙇

File: app/assets/scss/_search-page.scss (hunk 1 above)
Comment (domoscargin) [reply app/assets/scss/_search-page.scss:13]: Ah yes, thank you - I was going to bring this up. I kept this to match what the current list looks like, but I want to jettison and make the links underlined for precisely that reason.

Comment (lfdebrux) [reply app/assets/scss/_search-page.scss:13]: Jettison away 👾

**Review by lfdebrux — APPROVED** [review]

**Pull request merged**